ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -134,7 +106,39 @@ def _reload_features(metadata, features):
             log.debug("Updating ASM Remote Configuration: %s", features)
             # The order of this matters since 1click could reconfigure the AppSecProcessor
             # which the second checks
-            _appsec_1click_activation(tracer, features)
-            _appsec_rules_data(tracer, features)
+            self._appsec_1click_activation(features)
+            _appsec_rules_data(self.tracer, features)
+
+    def _appsec_1click_activation(self, features):
+        # type: (Union[Literal[False], Mapping[str, Any]]) -> None
+        if APPSEC_ENV in os.environ:
+            # no one click activation if var env is set
+            rc_appsec_enabled = asbool(os.environ.get(APPSEC_ENV))
+        elif features is False:
+            rc_appsec_enabled = False
+        else:
+            rc_appsec_enabled = features.get("asm", {}).get("enabled")
+
+        if rc_appsec_enabled is not None:
+            from ddtrace.appsec._constants import PRODUCTS
+            from ddtrace.internal.remoteconfig import RemoteConfig
 
-    return _reload_features
+            log.debug("Reloading Appsec 1-click: %s", rc_appsec_enabled)
```

Comment:
Can we not use "1-click" in code and logs as I don't think it's clear what this means. Can we say remote configuration instead?

Reply:
I'll do it in a new PR :+1: thanks!